NEWS.md

```diff
@@ -1,3 +1,6 @@
+## 6.2.0-SNAPSHOT 2020-11-23
```

Comment:
Has this version been released already? If not, why does it have a release date in the past?